ui/src/variables/actions/index.ts

```diff
@@ -69,6 +74,20 @@ const removeVariable = (id: string): RemoveVariable => ({
   payload: {id},
 })
 
+interface SelectValue {
```

Comment:
heh I just wrote something very similar. Let's merge yours first and then I'll consolidate after merging my PR.